modin/core/execution/ray/common/task_wrapper.py

```diff
@@ -39,10 +43,14 @@ def _deploy_ray_func(func, args):  # pragma: no cover
     ray.ObjectRef or list
         Ray identifier of the result being put to Plasma store.
     """
-    return func(**args)
+    logger = get_worker_logger(JOB_ID)
+    logger.info(f"START::PANDAS-API::deploy_ray_func.{func.__qualname__}")
+    res = func(**args)
```

Comment:
I see no exception handling, it would lead to logging unpaired `START` if any error is raised in `func`